### ISSUE TITLE
Pyrus doesn't work through http proxy

### ISSUE THREAD

**Issue by olofa** [pyrus/Pyrus#79]
Hi,

I'm using php 5.3.2 and pyrus 2.0.0a3. I have also tried with php 5.3.8.

I did:

pyrus set http_proxy fqdn:port

I run tcpdump on the http proxy server and this is what I get when trying to install a package:

GET /rest/p/pear2_net_transmitter/info.xml HTTP/1.0
Host: pear2.php.net
User-Agent: PEAR2_Pyrus/2.0.0a3/PHP/5.3.2
Accept:

Our squid3 proxy doesn't accept that. It wants to have it like this:

GET http://pear2.php.net/rest/p/pear2_net_transmitter/info.xml HTTP/1.0
Host: pear2.php.net
User-Agent: PEAR2_Pyrus/2.0.0a3/PHP/5.3.2
Accept:

I believe this is a bug in pyrus.

**Comment by saltybeagle** [comment]
Should be fixed now. Thanks your help tacking down the bug

**Comment by kemitix** [comment]
Hi, 

I'm getting this problem with the 2.0.0a4 release.

```
GET /rest/p/pyrus_developer/info.xml HTTP/1.0
Host: pyrus.net
User-Agent: Pyrus/2.0.0a4/PHP/5.5.9-1ubuntu4.3
Accept: 
Connection: close
```

**Comment by kemitix** [comment]
Further digging I've found that, although the problem is fixed with PEAR2_HTTP_Request-0.3.1, Pyrus is still bundled with 0.3.0. The github repo for PEAR2_HTTP_Request has no 0.3.1 tag, but its master branch does include the fix we need for this issue.

Pull request 138 adds the correct version of PEAR2_HTTP_Request.
